controllers/provider-gcp/charts/internal/gcp-infra/values.yaml

```diff
@@ -20,6 +20,11 @@ networks:
   pods: 100.96.0.0/11
   workers: 10.250.0.0/19
 # internal: 10.250.112.0/22
+  enableVPCFlowLogs: false
+  flowLog:
```

Comment:
Can you comment the whole block to make obvious that this is completely optional and nothing happens by default?

```suggestion
# flowLogs:
```